Makefile

```diff
@@ -224,6 +224,18 @@ endif
 $(KUBE_APISERVER) $(ETCD): ## install test asset kubectl, kube-apiserver, etcd
 	source ./scripts/fetch_ext_bins.sh && fetch_tools
 
+.PHONY: env-info # Temporary target to get additional logs in prow tests
+env-info:
+	@echo "Working dir is $(shell pwd)"
+	@echo "Working dir contents:"
+	ls -la
+	@echo "Root dir contents:"
+	ls -la /
```

Comment:
/hold

as discussed offline, please add an `|| echo ""` to avoid failing the job if we are unable to list root